src/main/java/org/example/homework6/MyMethods.java

```diff
@@ -0,0 +1,60 @@
+package org.example.homework6;
+
+import java.util.Scanner;
+
+public class MyMethods {
```

Comment:
If you extract all this logic to a separate class, MyMethods is not the best option